spec/acceptance/nodesets/ubuntu-16.yml

```diff
@@ -0,0 +1,12 @@
+---
+HOSTS:
+  ubuntu-16:
+    roles:
+      - master
+    platform: ubuntu-1604-x86_64
+    box:  puphpet/ubuntu1604-x64
+    box_url: https://atlas.hashicorp.com/puphpet/boxes/ubuntu1604-x64
```

Comment:
FYI - This is the config that (if you trust uploader) built this box: https://github.com/puphpet/bento/blob/master/ubuntu-16.04-amd64.json